tools/build_variables.py

```diff
@@ -140,6 +140,7 @@
     "torch/csrc/jit/passes/subgraph_rewrite.cpp",
     "torch/csrc/jit/passes/utils/subgraph_utils.cpp",
     "torch/csrc/jit/passes/utils/memory_dag.cpp",
+    "torch/csrc/jit/passes/freeze_madule.cpp",
```

Comment:
Typo: madule

Reply:
fixed!

Reply:
You probably forgot to push the fixed version :)